index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <title>Canvas</title>
+</head>
+<body>
+  <div class="body-wrapper">
+    <header class="header">
```

Comment:
This "header" class seems useless, since for the essential elements like "body", "header", "footer" which are used on the page only once, styles are applied in the css files directly by the tag name